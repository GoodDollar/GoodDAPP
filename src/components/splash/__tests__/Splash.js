import React from 'react'

// Note: react-test-renderer should be imported after react
import renderer from 'react-test-renderer'

import { withThemeProvider } from '../../../__tests__/__util__'
import ImportedSplash from '../Splash'

const Splash = withThemeProvider(ImportedSplash)

describe('Splash', () => {
  it('matches snapshot', async () => {
    let component

    // eslint-disable-next-line require-await
    await renderer.act(async () => (component = renderer.create(<Splash />)))
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
