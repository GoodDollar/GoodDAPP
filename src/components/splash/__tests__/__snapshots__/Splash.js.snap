// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash matches snapshot 1`] = `
<div
  className="css-view-1dbjc4n"
  style={
    Object {
      "WebkitBoxFlex": 1,
      "WebkitFlexBasis": "0%",
      "WebkitFlexGrow": 1,
      "WebkitFlexShrink": 1,
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "msFlexNegative": 1,
      "msFlexPositive": 1,
      "msFlexPreferredSize": "0%",
    }
  }
>
  <div
    className="css-view-1dbjc4n"
    data-name="viewWrapper"
    style={
      Object {
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "backgroundImage": "linear-gradient(to bottom, #00AFFF, #2DC0F7, #28C0EF, #23C0E7, #1EC1DF, #19C1D7, #14C1CF, #0FC2C7, #0FC2C7, #0AC2BF, #05C2B7, #00C3AF)",
        "display": "flex",
        "flexDirection": "column",
        "flexGrow": 1,
        "maxHeight": "844px",
        "msFlexDirection": "column",
        "msFlexPositive": 1,
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
        "position": "relative",
        "width": "100%",
      }
    }
  >
    <div
      className="css-view-1dbjc4n"
      style={
        Object {
          "WebkitAlignItems": "center",
          "WebkitBoxAlign": "center",
          "WebkitBoxFlex": 1,
          "WebkitBoxPack": "center",
          "WebkitFlexBasis": "0%",
          "WebkitFlexGrow": 1,
          "WebkitFlexShrink": 1,
          "WebkitJustifyContent": "center",
          "WebkitTransform": "rotateY(180deg)",
          "alignItems": "center",
          "backgroundColor": "rgba(0,0,0,0.00)",
          "borderBottomLeftRadius": "5px",
          "borderBottomRightRadius": "5px",
          "borderTopLeftRadius": "5px",
          "borderTopRightRadius": "5px",
          "flexBasis": "0%",
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "msFlexAlign": "center",
          "msFlexNegative": 1,
          "msFlexPack": "center",
          "msFlexPositive": 1,
          "msFlexPreferredSize": "0%",
          "paddingBottom": "16px",
          "paddingLeft": "12px",
          "paddingRight": "12px",
          "paddingTop": "16px",
          "position": "relative",
          "transform": "rotateY(180deg)",
        }
      }
    >
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "backgroundImage": "url(splashWaves.svg)",
            "backgroundRepeat": "repeat-y",
            "height": "100%",
            "opacity": 0.1,
            "position": "absolute",
            "width": "100%",
          }
        }
      />
      <div
        className="css-view-1dbjc4n"
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxFlex": 1,
            "WebkitBoxOrient": "vertical",
            "WebkitBoxPack": "center",
            "WebkitFlexDirection": "column",
            "WebkitFlexGrow": 1,
            "WebkitJustifyContent": "center",
            "WebkitTransform": "rotateY(180deg)",
            "alignItems": "stretch",
            "flexDirection": "column",
            "flexGrow": 1,
            "justifyContent": "center",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexPack": "center",
            "msFlexPositive": 1,
            "overflowX": "hidden",
            "overflowY": "hidden",
            "transform": "rotateY(180deg)",
          }
        }
      >
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "WebkitAlignItems": "stretch",
              "WebkitBoxAlign": "stretch",
              "WebkitBoxDirection": "normal",
              "WebkitBoxOrient": "vertical",
              "WebkitBoxPack": "justify",
              "WebkitFlexDirection": "column",
              "WebkitJustifyContent": "space-between",
              "alignItems": "stretch",
              "flexDirection": "column",
              "justifyContent": "space-between",
              "msFlexAlign": "stretch",
              "msFlexDirection": "column",
              "msFlexPack": "justify",
            }
          }
        >
          <div
            className="css-text-901oao"
            dir="auto"
            letterSpacing={0.13}
            style={
              Object {
                "borderBottomColor": "rgba(0,0,0,1.00)",
                "borderBottomStyle": "solid",
                "borderBottomWidth": "2px",
                "color": "rgba(255,255,255,1.00)",
                "direction": "ltr",
                "fontFamily": "Roboto",
                "fontSize": "26px",
                "fontWeight": "700",
                "letterSpacing": "0.13px",
                "lineHeight": "32px",
                "marginBottom": "10px",
                "marginLeft": "auto",
                "marginRight": "auto",
                "paddingBottom": "8px",
                "paddingLeft": "25px",
                "paddingRight": "25px",
                "textAlign": "center",
                "textDecoration": "none",
                "textTransform": "none",
              }
            }
          >
            GoodDollar Demo
          </div>
          <div
            className="css-text-901oao"
            dir="auto"
            letterSpacing={0.24}
            style={
              Object {
                "color": "rgba(255,255,255,1.00)",
                "direction": "ltr",
                "fontFamily": "Roboto",
                "fontSize": "16px",
                "fontWeight": "500",
                "letterSpacing": "0.24px",
                "lineHeight": "22px",
                "textAlign": "center",
                "textDecoration": "none",
                "textTransform": "none",
              }
            }
          >
            All G$ coins in the demo
are for test purposes only.
Once all feedback is incorporated,

            <span
              className="css-text-901oao css-textHasAncestor-16my406"
              dir="auto"
              letterSpacing={0.24}
              style={
                Object {
                  "color": "rgba(255,255,255,1.00)",
                  "direction": "ltr",
                  "fontFamily": "Roboto",
                  "fontSize": "16px",
                  "fontWeight": "700",
                  "letterSpacing": "0.24px",
                  "lineHeight": "22px",
                  "textAlign": "center",
                  "textDecoration": "none",
                  "textTransform": "none",
                }
              }
            >
              all demo G$ coins will be deleted.
            </span>
          </div>
        </div>
        <div
          className="css-view-1dbjc4n"
          style={
            Object {
              "height": "550px",
              "marginBottom": "-120px",
              "marginTop": "-75px",
              "width": "100%",
            }
          }
        />
        <div
          className="css-text-901oao"
          dir="auto"
          style={
            Object {
              "color": "rgba(12,38,61,1.00)",
              "direction": "ltr",
              "fontFamily": "Roboto",
              "fontSize": "16px",
              "fontWeight": "500",
              "lineHeight": "22px",
              "textAlign": "center",
              "textDecoration": "none",
              "textTransform": "none",
            }
          }
        >
          Demo 
          V
          v0
        </div>
      </div>
    </div>
  </div>
</div>
`;
